src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncSpec.cs

```diff
@@ -66,11 +65,17 @@ public void A_Flow_with_SelectAsync_must_produce_task_elements_in_order()
         {
             var c = this.CreateManualSubscriberProbe<int>();
             Source.From(Enumerable.Range(1, 50))
-                .SelectAsync(4, i => Task.Run(()=>
+                .SelectAsync(4, i =>
                 {
-                    Thread.Sleep(ThreadLocalRandom.Current.Next(1, 10));
-                    return i;
-                }))
+                    if (i%3 == 0)
```

Comment:
You definitely should set this setting `SPACE_AROUND_MULTIPLICATIVE_OP` in Resharper to `true`